### ISSUE TITLE
About the matrix of metacorrection generation step

### ISSUE THREAD

**Issue by CinKKKyo** [cyang-cityu/MetaCorrection#11]
Hi, I do have a problem when I try to reimplement you guys ideas, I saw the target labels were used in the meta-learning step? (as shown in "train_adv.py", line281), but I saw in the paper, we shouldn't use the target labels in the training steps. Is any misunderstanding for me? Could you help me to figure it out?Thanks a lot~
